packages/reassure-compare/src/utils/format.ts

```diff
@@ -100,7 +100,7 @@ function getRenderCountSymbols(entry: CompareEntry) {
 
 export function formatMetadata(metadata?: PerformanceMetadata) {
   if (metadata?.branch && metadata?.commitHash) {
```

Comment:
Formatting branch + commit hash should be independent of formatting date:

```
function formatCommitMetadata() {
  if (metadata?.branch && metadata?.commitHash) {
    return `${metadata.branch} (${metadata.commitHash})
  }

 return metadata?.branch || metadata?.commitHash || '(unknown)';
}

function formatMetadata() {
  let result = formatCommitMetadata();
  if (metadata.creationDate) {
    result += ` - ${metadata.creationDate}`
  }

  return result;
}
```

Reply:
> Formatting branch + commit hash should be independent of formatting date:
> 
> ```
> function formatCommitMetadata() {
>   if (metadata?.branch && metadata?.commitHash) {
>     return `${metadata.branch} (${metadata.commitHash})
>   }
> 
>  return metadata?.branch || metadata?.commitHash || '(unknown)';
> }
> 
> function formatMetadata() {
>   let result = formatCommitMetadata();
>   if (metadata.creationDate) {
>     result += ` - ${metadata.creationDate}`
>   }
> 
>   return result;
> }
> ```

Alright I can resolve this